ruled_labels/[DRAFT]specs_monorepo.yaml

```diff
@@ -29,21 +32,24 @@ labels:
   - name: A3-backport
     description: Pull request is already reviewed well in another branch.
     color: F8C5E7
-  - name: C0-easy
+  - name: C1-mentor
+    description: A task where a mentor is available. Please indicate in the issue who the mentor could be.
+    color: "5319e7"
+  - name: C2-ready_for_review
+    description: A PR opened by a contributor is ready for review.
+    color: "249899"
```

Comment:
We should leave it up to the project to define the status of the PR/Issue, not to the labels

Reply:
generally agree, but what if a contributor picked up an issue and opened a PR for it. Should they just assign the PR to a project and wait? I've added this label bc of the [new contribution guideline](https://www.notion.so/paritytechnologies/new-polkadot-sdk-contribution-guideline-79adc397b1674983b341eceeb091329c?pvs=4), as we had something similar before. Also with that, we could filter and find PRs that were submitted by contributors only.

Reply:
Actually github supports to open prs as "draft". If they are draft, they are not ready. If they are not draft, they are ready.